src/components/v5/shared/Button/types.ts

```diff
@@ -82,6 +84,21 @@ export interface CloseButtonProps extends CommonButtonProps {
   className?: string;
 }
 
+export interface ActionButtonProps extends ButtonProps {
+  actionType: ActionTypes;
+  isLoading?: boolean;
+  confirmText?: any;
+  error?: string;
+  onConfirmToggled?: (...args: any[]) => void;
+  onSuccess?: (result: any) => void;
+  onError?: (error: any) => void;
+  submit?: string;
+  success?: string;
+  text?: MessageDescriptor | string;
+  transform?: ActionTransformFnType;
+  values?: any;
```

Comment:
please avoid using `any`

Reply:
This are used by sagas, so we don't exactly know what will be passed here. It was copied from previous app.